src/presentation/http/router/note.test.ts

```diff
@@ -58,39 +60,33 @@ describe('Note API', () => {
       const expectedStatus = 200;
       const correctID = 'Pq1T9vc23Q';
 
-      const expectedNote = {
-        'id': 2,
-        'publicId': 'Pq1T9vc23Q',
-        'creatorId': 1,
-        'content': null,
-        'createdAt': '2023-10-16T13:49:19.000Z',
-        'updatedAt': '2023-10-16T13:49:19.000Z',
-      };
-
       const response = await global.api?.fakeRequest({
         method: 'GET',
         url: `/note/${correctID}`,
       });
 
       expect(response?.statusCode).toBe(expectedStatus);
 
-      expect(response?.json()).toStrictEqual(expectedNote);
+      expect(response?.json()).toStrictEqual({
+        'note': {
+          'id': 2,
+          'publicId': 'Pq1T9vc23Q',
+          'creatorId': 1,
+          'content': null,
+          'createdAt': '2023-10-16T13:49:19.000Z',
+          'updatedAt': '2023-10-16T13:49:19.000Z',
+        },
+        'accessRights': {
+          'canEdit': false,
+        },
+      });
```

Comment:
maybe make expectedResponse with access rights and note in it as it was with the expectedNote?